.github/workflows/test.yaml

```
@@ -320,8 +320,16 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v6
 
+      - id: Check if nix files changed
```

actionlint (1.7.10)

invalid step ID "Check if nix files changed". step ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)